src/OAuth/Config/Environment.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Penneo\SDK\OAuth\Config;
+
+final class Environment
+{
+    public const SANDBOX = 'sandbox';
+    public const PRODUCTION = 'production';
+}
```

Comment:
Could this be just an enum instead of a class?

Reply:
Enums were introduced in php 8.1.0, but the minimum version for the SDK is 7.2, so I am afraid it cannot (without hacks)